docs/UserGuide.md

```diff
@@ -74,27 +74,14 @@ Here is a sequence of example commands you can try:<br>
 
 ## Features
```

Comment:
Missing the table and command breakdown example (think we want to model after the UG example we saw ytd). Can refer to the pdf shared in the group (I also asked Clara to share the link to this model UG there) 

Specifically, for features:

-brief explanation on our features
-"Listed below are the common components of a command"
-(convert the notes box to table below)

-table
>>component (command word, prefix, parameter)
>>description

-Example
-"add t/TITLE by/DEADLINE m/MILESTONE [c/CONTACTS] [d/DESCRIPTION]"
-breakdown the example
-<Note box>: square brackets mean optional prefix-parameter pairs